packages/api/knex_migrations/20181127110637_create_async_operations_table.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const { DynamoDbScanQueue } = require('@cumulus/common/aws');
+
+function dynamoDateToPgDate(dynamoDate) {
+  return (new Date(Number(dynamoDate))).toISOString();
+}
+
+function dynamoRecordToPgRecord(dynamoRecord) {
+  const pgRecord = { id: dynamoRecord.id.S };
+
+  if (dynamoRecord.status) pgRecord.status = dynamoRecord.status.S;
+  if (dynamoRecord.output) pgRecord.output = dynamoRecord.output.S;
+  if (dynamoRecord.taskArn) pgRecord.task_arn = dynamoRecord.taskArn.S;
+
+  if (dynamoRecord.createdAt) {
+    pgRecord.created_at = dynamoDateToPgDate(dynamoRecord.createdAt.N);
+  }
+
+  if (dynamoRecord.updatedAt) {
+    pgRecord.updated_at = dynamoDateToPgDate(dynamoRecord.updatedAt.N);
+  }
+
+  return pgRecord;
+}
+
+exports.up = async (knex) => {
+  await knex.schema.createTable(
+    'async_operations',
+    (table) => {
+      table.uuid('id').primary().notNull();
+      table.timestamps(false, true);
+      table.text('output').nullable();
+      table.enu(
+        'status',
+        ['RUNNING', 'SUCCEEDED', 'RUNNER_FAILED', 'TASK_FAILED']
+      ).notNull();
+      table.string('task_arn').nullable();
```

Comment:
Should we include a meta.json in all tables, or just the ones that obviously relate to user data concerns?